ddtrace/contrib/rq/__init__.py

```diff
@@ -0,0 +1,236 @@
+"""
+The RQ integration will trace your jobs.
+
+Worker Usage
+~~~~~~~~~~~~
+
+``ddtrace-run`` can be used to easily trace your workers::
+
+    DD_WORKER_SERVICE_NAME=myworker ddtrace-run rq worker
+
+
+See https://github.com/DataDog/trace-examples/tree/kyle-verhoog/rq/python/rq for a working example.
```

Comment:
This link will be updated once the example is finalized and merged into the `trace-examples` repo